### PR TITLE
tools: use yarn script in scaffolder

### PR DESCRIPTION
**Pull request by dschaller**
### Description

give preference to yarn script in scaffolder and fallback to the global yarn installation.

### Testing Performed

manual

### CODE REVIEW

File: tools/scaffolding/scaffolder.go [hunk 1]
```diff
@@ -79,6 +79,23 @@ func determineGoPath() string {
...
+		log.Fatal(err)
+	}
+	yarnScript := filepath.Join(path, "..", "..", "build", "bin", "yarn.sh")
+	scriptPath, err := exec.LookPath(yarnScript)
+	if err != nil {
+		path, err := exec.LookPath("yarn")
+		if err != nil {
+			log.Fatal("could not find yarn executable to use")
+		}
+		return strings.TrimSpace(path)
```
Comment (danielhochman) [inline tools/scaffolding/scaffolder.go:94]: what's with the TrimSpace?

Comment (dschaller) [reply tools/scaffolding/scaffolder.go:94]: I was following the pattern from `determineGoPath` which I assume did it to be safe

Comment (danielhochman) [reply tools/scaffolding/scaffolder.go:94]: that just comes from the fact that it's executing a command that returns with a newline. i don't think any of your commands do that.

**Review by danielhochman — APPROVED** [review]
i think you can remove trimspace otherwise lgtm

**Review by miguelmolina95 — APPROVED** [review]

**Pull request merged**